scripts/mod/file2alias.c

```diff
@@ -1364,6 +1364,13 @@ static int do_mhi_entry(const char *filename, void *symval, char *alias)
 {
 	DEF_FIELD_ADDR(symval, mhi_device_id, chan);
 	sprintf(alias, MHI_DEVICE_MODALIAS_FMT, *chan);
+	return 1;
```

Comment:
this seems to be modifying an unrelated function?

Reply:
not sure where this came from - maybe need a rebase?

Reply:
THis is just an artifact of the patching process - it is inserting our addition between the:
sprintf(alias, MHI_DEVICE_MODALIAS_FMT, *chan); in do_mhi_enrtry and the return 1;

So we are adding a return +1 here, but the original one is being used in our inserted function.

Reply:
what I mean is, that before this patch the `do_mhi_entry()` function had an empty line before `return 1;` Your patch removes it for some reason.